labs/lab 3/Helloworld.html

```diff
@@ -0,0 +1,33 @@
+<html> 
+<body> 
+
+<script> 
+	let msg = 'hello world'; 
+	
+	console.log(msg); 
+	
+	function fToC(f){ 
+		return (f - 32) * 5 / 9; 
+	} 
+	console.log(fToC(32)); 
+	console.log(fToC(212)); 
+	console.log(fToC(68)); 
+
+	function cToK(c){ 
+		return c + 273.15	
+	} 
+	console.log(cToK(0)); 
+	console.log(cToK(100)); 
+	console.log(cToK(20)); 
+	
+	function fToK(k){ 
+		return fToC + cToK - 273.15
```

Comment:
The input variable should be named `f` since that's what you get.  Then you need to actually invoke (call) your `fToC` and `cToK` functions in order to transform the input.  This function doesn't work in its current state.